tsconfig-aot.json

```diff
@@ -14,7 +14,7 @@
     "pretty": true,
     "stripInternal": true,
     "skipLibCheck": true,
-    "outDir": "dist",
+    "outDir": "../rpx-xui-webapp/node_modules/@hmcts/ccd-case-ui-toolkit/dist",
```

Comment:
This looks like something you set up for local testing, rather than something that should be committed.